backend/module/eCampLib/src/Util/IdGenerator.php

```diff
@@ -2,14 +2,10 @@
 
 namespace eCamp\Lib\Util;
 
-use RuntimeException;
+use Laminas\Math\Rand;
 
 class IdGenerator {
     public static function generateRandomHexString(int $length): string {
-        try {
-            return bin2hex(random_bytes($length / 2));
-        } catch (\Exception $e) {
-            throw new RuntimeException($e);
-        }
+        return bin2hex(Rand::getBytes($length / 2));
```

Comment:
@carlobeltrame this is still using `random_bytes`, but wrapped in `Laminas\Math\Rand`, correct?

Reply:
Yes, the internal implementation is the same for now. But we now delegate to the framework the responsibility to keep up to date with the latest and greatest PHP has to offer for this job.